cob_cartesian_controller/src/simple_cartesian_interface/simple_cartesian_interface.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+import rospy
+import threading
+import actionlib
+
+from tf.transformations import *
+from geometry_msgs.msg import Pose
+from cob_cartesian_controller.msg import CartesianControllerAction, CartesianControllerGoal
+from cob_cartesian_controller.msg import Profile
+
+
+_transform_listener = None
+_transform_listener_ock = threading.Lock()
+
+_action_client = None
+_action_client_lock = threading.Lock()
+
+def move_lin(pose_goal, frame_id, profile, rotate_only=False):
+    client = actionlib.SimpleActionClient('cartesian_trajectory_action', CartesianControllerAction)
+    rospy.logwarn("Waiting for ActionServer...")
+    success = client.wait_for_server(rospy.Duration(2.0))
+    if(not success):
+        return (success, "ActionServer not available within timeout")
+    
+    goal = CartesianControllerGoal()
+    goal.name = "move_lin"
+    goal.move_lin.pose_goal = pose_goal
+    goal.move_lin.frame_id = frame_id
+    goal.move_lin.rotate_only = rotate_only
+    goal.move_lin.profile = profile
+    print goal
+    
+    client.send_goal(goal)
```

Comment:
If the action server is always used with send_goal and wait_for_result we should think about implementing a "simple" service that blocks until the movement has finished. Depends on the use cases.